app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/edit.hbs

```diff
@@ -0,0 +1,273 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuHeading>
+        Bewerk mandataris
+      </AuHeading>
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-top-tiny">
+        {{@model.person.givenName}}
+        {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Naam</:label>
+            <:content>
+              {{@model.person.givenName}}
+              {{@model.person.familyName}}
+            </:content>
+          </Item>
+        </Grid>
+      </:card>
+    </DataCard>
+    <form
+      id="edit-mandatory-form"
+      aria-label="edit-mandatory-form"
+      {{on "submit" (perform this.save)}}
+    >
+      <div class="au-o-flow au-o-flow--large">
+        <EditCard>
+          <:title>Positie</:title>
+          <:card as |Grid|>
+            <Grid as |Item|>
+              <Item>
+                {{! TODO: Put in a disable PowerSelect }}
+                <:label>Type</:label>
+                <:content>Mandaat</:content>
+              </Item>
+              <Item>
+                <:label>Naam Eredienst</:label>
+                <:content>
+                    {{!-- TODO: Make it show the administrative unit's name --}}
+                    <PowerSelect
+                      @disabled={{true}}
+                      @selected={{this.model.administrativeUnit}}
```

Comment:
this has been aliased in the controller so I would use the aliased version here. Same for most of the others in the template :smile: